src/pynwb/form/build/map.py

```diff
@@ -200,6 +200,10 @@ def __init__(self, **kwargs):
         self.__carg2spec = dict()
         self.__map_spec(spec)
 
+
+    def hack_get_subspec_values(self, *args, **kwargs):
+        return self.__get_subspec_values(*args, **kwargs)
```

Comment:
@nicain I suggest to mark this method as "private" changing to `_hack_get_subspec_values`